### PR TITLE
Change kits to templates

### LINKED ISSUE #111: change kits -> templates
e.g. https://hack.aragon.org/docs/kits-intro

See: https://forum.aragon.org/t/on-aragon-naming/203

### PR DESCRIPTION
**Pull request by 0xGabi**
closes #111 

- [x] Deprecate kits in aragonCLI https://github.com/aragon/aragon-cli/issues/404

### CODE REVIEW

File: docs/cli-main-commands.md [hunk 1]
```diff
@@ -57,6 +57,8 @@ However, when **making changes to the background script** of your app, a refresh
 
 The [React boilerplate](https://github.com/aragon/aragon-react-boilerplate) supports serving your app using HTTP.
 
+> **Note**<br>
+> The `kits` has been deprecated and `templates` should be used instead. You may find the `kits` notation in some places while we made the transition.
```
Comment (kernelwhisperer) [inline docs/cli-main-commands.md]:
```suggestion
> The `kits` has been deprecated and `templates` should be used instead. You may find the `kits` notation in some places while we make the transition.
```

File: docs/guides-custom-deploy.md [hunk 2]
```diff
@@ -10,12 +10,13 @@ We assume you have a general understanding of Aragon stack.
 
 In order to get started you will need the latest version of the CLI installed so open a terminal and enter the following:
 
-
 ```sh
 npm install -g @aragon/cli
 
 ```
 
+av
```
Comment (kernelwhisperer) [inline docs/guides-custom-deploy.md]:
```suggestion
```

File: docs/templates-intro.md [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+Templates can also be thought of as **DAO templates** as every template can create a DAO with specific settings for an organization type. The two DAO configurations that one can choose from when using the [Aragon client](http://app.aragon.org) correspond to the beta templates ([Democracy](https://github.com/aragon/dao-kits/blob/ce62d132d944951dc200df8aa74e42db8e70a094/kits/democracy/contracts/DemocracyKit.sol) and [Multisig](https://github.com/aragon/dao-kits/blob/ce62d132d944951dc200df8aa74e42db8e70a094/kits/multisig/contracts/MultisigKit.sol)) available in the [DAO templates repository](https://github.com/aragon/dao-kits).
+
+## Getting started with templates using the aragonCLI
+
+The [aragonCLI](/docs/cli-intro.html) (>= v4.1.0) supports using templates to create a DAO to interact with the apps being developed.
+
+To quickly get started developing your own templates:
+
+```sh
+npx create-aragon-app app react-template
```
Comment (kernelwhisperer) [inline docs/templates-intro.md]:
```suggestion
npx create-aragon-app app
```

Since we are merging `react` and `react-kit`.

**Review by kernelwhisperer — APPROVED** [review]

File: docs/glossary.md [hunk 4]
```diff
@@ -16,7 +16,7 @@ sidebar_label: Glossary
 - [**aragonUI**](/docs/aragonui-intro.html): Aragon-native toolkit of UI components for decentralized apps.
 - [**Aragon client**](http://app.aragon.org): Client to interact with aragonOS-powered decentralized organizations. It implements a signer with transaction pathing, notifications and a sandboxed environment for aragonOS-based apps using aragonAPI.
 - **DAO**: Stands for Decentralized Autonomous Organization. Here, a DAO means a set of aragonOS smart contracts.
-- [**DAO Kit**](/docs/kits-intro.html): Smart contracts that can be used as templates for creating pre-configured DAOs in just one transaction.
+- [**DAO Template**](/docs/templates-intro.html): Smart contracts that can be used as templates for creating pre-configured DAOs in just one transaction.
```
Comment (sohkai) [inline docs/glossary.md:19]: Could we keep an entry for "DAO Kits" here that say they're deprecated for "DAO Templates"?

File: docs/templates-intro.md [hunk 5]
```diff
@@ -0,0 +1,49 @@
...
+All templates follow a similar structure:
+
+1. Use a [DAOFactory](https://github.com/aragon/aragonOS/blob/4.0.1/contracts/factory/DAOFactory.sol) to create a DAO.
+2. Assign the template contract the necessary permissions in the DAO needed for installation (usually only `APP_MANAGER_ROLE`).
+3. Create app proxy instances for all the apps (`dao.newAppInstance(...)`).
+4. Initialize apps (`app.initialize(...)`).
+5. Set up permissions for the apps and the DAO.
+6. Clean up the permissions temporarily assigned to the template.
+
+> **Note**<br>
```
Comment (sohkai) [inline docs/templates-intro.md:48]: Could we move this note to be right before the "Getting started with templates using the aragonCLI" so it's more visible?

**Review by sohkai — COMMENTED** [review]
Awesome, thanks for doing this @0xGabi!

I just added a few suggestions :).

**Pull request merged**